### PR TITLE
INC-871: Use simpler & clearer reasons for posting automated incentive reviews

### PR DESCRIPTION
**Pull request by ushkarev**
…on admission, readmission and transfer (irrespective of whether the level was not identical before)

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt [hunk 1]
```diff
@@ -1279,7 +1280,14 @@ class PrisonerIepLevelReviewServiceTest {
       reason = reason
     ),
     occurredAt = Instant.now(),
-    description = "A prisoner has been received into prison"
+    description = "A prisoner has been received into a prison with reason: " + when (reason) {
```
Comment (ushkarev) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt:1283]: might as well use the real event descriptions

File: src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt [hunk 2]
```diff
@@ -353,6 +354,12 @@ class PrisonerIepLevelReviewService(
     }
   }
 
+  private fun getReviewCommentForEvent(prisonOffenderEvent: HMPPSDomainEvent) = when (prisonOffenderEvent.additionalInformation.reason) {
+    "NEW_ADMISSION", "READMISSION" -> "Default level assigned on arrival"
+    "TRANSFERRED" -> "Level transferred from previous establishment"
```
Comment (ushkarev) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt:359]: at this point, we're choosing to not explain why the level may have been changed

**Review by neilmendum — APPROVED** [review]
LGTM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt [hunk 3]
```diff
@@ -1289,7 +1297,7 @@ class PrisonerIepLevelReviewServiceTest {
       categoriesChanged = categoriesChanged,
     ),
     occurredAt = Instant.now(),
-    description = "A prisoner has been received into prison"
+    description = "A prisoner record has been updated"
```
Comment (xoen) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt:1300]: Ops, I got the wrong one, good spot!

**Pull request merged**